LICENSE

```diff
@@ -1,3 +1,5 @@
+Copyright 2016-2024 ClickHouse, Inc.
```

Comment:
this is not the right place to add the copyright, this file should be kept as-is. 

usually, copyright info should be added to each file's license header, and the NOTICE file.

Reply:
This our convention in ClickHouse i will check with legal